package.json

```diff
@@ -4,22 +4,30 @@
   "description": "Identity Provider using jwt-redirection",
   "main": "server.js",
   "scripts": {
-    "start": "set DEV=true&& start mhub-server&& node server.js",
-    "lint": "eslint \"*.js\"",
-    "lint:fix": "eslint --fix \"*.js\""
+    "clean": "rimraf public",
```

Comment:
Just for my own edification: what does this do?

Reply:
deletes the directory `public`. the name kind of comes from the linux command `rm -rf`